addons/web_tour/static/src/debug/debug_manager.js

```diff
@@ -12,14 +12,15 @@ const { Component } = owl;
 patch(ActionContainer.prototype, 'web_tour.webclient', {
     setup() {
         this._super(...arguments);
```

Comment:
Component.setup() takes no arguments ;-)

Reply:
True, but we tend to keep the ...arguments anyway for consistency I believe